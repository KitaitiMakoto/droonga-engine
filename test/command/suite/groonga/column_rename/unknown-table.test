{
  "type": "column_rename",
  "dataset": "Droonga",
  "body": {
    "table" : "Unknown"
  }
}
