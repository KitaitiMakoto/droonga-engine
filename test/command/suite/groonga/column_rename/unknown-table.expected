{
  "inReplyTo": "request-id",
  "statusCode": 200,
  "type": "column_rename.result",
  "body": [
    [
      -22,
      0.0,
      0.0,
      "table not found: <Unknown>"
    ],
    false
  ]
}
